test_scripts/Polices/Policies_Security/001_P_ATF_Policies_MemoryFor_PTS.lua

```diff
@@ -0,0 +1,95 @@
+-- UNREADY---
+-- Iliana as I understand, we need create a function witch will be return How much memory Policy Manager have.
+-- I have problems with it. I just send sdl_preloaded json size near 1 mb, and done PTU with it. Please help me with this function for check avaliable memory.
+-- Thanks a lot.
+
+-- Requirement summary:
+-- [General] Policies enough memory for PTS
+ 
+-- Description:
+-- Before sending PTS via OnSystemRequest to mobile app, SDL checks to have available more than 1 Mbytes of allocated memory RAM.
+-- Policies Manager must have enough memory to hold a Policy Table Snapshot in memory. 
+-- Please rename attached ptu_1mb.json and copy this file on this way /tmp/fs/mp/images/ivsu_cache/,
+-- And for run SDL use attached sdl_preloaded_pt.
+
+-- Performed steps
+--       1. MOB-SDL - register application
+--       2. SDL - start PTU
+--       3. Before OnSystemRequest SDL checks to have available more than 1 Mbytes of allocated memory RAM
+--       3. Finish PTU.
+
+
+ --[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ Local preparing ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+commonFunctions:newTestCasesGroup("Preconditions")
+
+--[[ Preconditions ]]
+commonFunctions:userPrint(33, "Precondition")
```

Comment:
Please use commonFunctions:newTestCasesGroup("Preconditions") as described in template. We try to have these functions unified.